.bazelrc

```diff
@@ -49,5 +49,25 @@ build:clang-msan --define tcmalloc=disabled
 build:clang-msan --copt -fsanitize-memory-track-origins=2
 build:clang-msan --linkopt -fuse-ld=lld
 
+# Mac ASAN/UBSAN
+build:mac-asan --define ENVOY_CONFIG_ASAN=1
+build:mac-asan --copt -D__SANITIZE_ADDRESS__
+build:mac-asan --copt -fsanitize=address,undefined
+build:mac-asan --linkopt -fsanitize=address,undefined
+build:mac-asan --copt -fno-sanitize=vptr
+build:mac-asan --linkopt -fno-sanitize=vptr
+build:mac-asan --copt -fno-sanitize-recover=all
+build:mac-asan --linkopt -ldl
+build:mac-asan --define tcmalloc=disabled
+build:mac-asan --build_tag_filters=-no_asan
+build:mac-asan --test_tag_filters=-no_asan
+build:mac-asan --define signal_trace=disabled
+build:mac-asan --copt -DADDRESS_SANITIZER=1
+build:mac-asan --test_env=ASAN_SYMBOLIZER_PATH
+build:mac-asan --test_env=ASAN_OPTIONS=handle_abort=1:allow_addr2line=true:check_initialization_order=true:strict_init_order=true
+# Workaround, see https://github.com/bazelbuild/bazel/issues/6932
+build:mac-asan --copt -Wno-macro-redefined
+build:mac-asan --copt -D_FORTIFY_SOURCE=0
```

Comment:
Could you use the config expansion instead of copy & pasting whole configs (same for `clang-asan`)? It makes it hard to notice what's common and what's unique to this config.

It should read like this:
```
# Clang ASAN/UBSAN
build:clang-asan --config=asan
build:clang-asan --copt -D__SANITIZE_ADDRESS__
build:clang-asan --test_env=ASAN_SYMBOLIZER_PATH
build:clang-asan --test_env=ASAN_OPTIONS=handle_abort=1:allow_addr2line=true:check_initialization_order=true:strict_init_order=true
build:clang-asan --linkopt -fuse-ld=lld

# Mac ASAN/UBSAN
build:mac-asan --config=asan
build:mac-asan --copt -D__SANITIZE_ADDRESS__
build:mac-asan --test_env=ASAN_SYMBOLIZER_PATH
build:mac-asan --test_env=ASAN_OPTIONS=handle_abort=1:allow_addr2line=true:check_initialization_order=true:strict_init_order=true
# Workaround, see https://github.com/bazelbuild/bazel/issues/6932
build:mac-asan --copt -Wno-macro-redefined
build:mac-asan --copt -D_FORTIFY_SOURCE=0
```
Note: You can probably move `--copt -D__SANITIZE_ADDRESS__` and `--test_env=ASAN_OPTIONS=...` to the common `asan` config as well.